airflow/utils/log/file_task_handler.py

```diff
@@ -190,6 +191,15 @@ def _read(self, ti, try_number, metadata=None):
                 response.raise_for_status()
 
                 log += '\n' + response.text
+            except HTTPStatusError as e:
+                if e.response.status_code == 403:
+                    log += f"*** Failed to fetch log file from worker With 403, Forbidden error. {str(e)}\n"
```

Comment:
```suggestion
                    log += f"*** Failed to fetch log file from worker With {str(e)}\n"
```

I'd guess (but haven't tested) that stringifying the error would show "403 Forbidden" -- and if so we don't need to say that again.

Reply:
I put it a bit differently actually - before raising the status to make it simpler. I think it's still worth to add the str(err) in next line anyway as it might contain potentially some more diagnostics messages (for example if the forbidden is returned rather by a proxy than worker - it might contain extra message).